drivers/CAN.h

```diff
@@ -288,13 +297,15 @@ class CAN : private NonCopyable<CAN> {
 
     static void _irq_handler(uint32_t id, CanIrqType type);
 
+#if !defined(DOXYGEN_ONLY)
 protected:
     virtual void lock();
     virtual void unlock();
     can_t               _can;
     Callback<void()>    _irq[IrqCnt];
     PlatformMutex       _mutex;
 };
+#endif
```

Comment:
this should be oneline up as it would enclose protected members rather than entire class (assuming line 307 is about class declaration end)

Reply:
Yes, big mistake.. Will fix it